### DISCUSSION TITLE
Showcasing a New Version of Stride Docs

### DISCUSSION THREAD

**Discussion by VaclavElias** [dotnet/docfx#8679] [Show and tell]
Hi everyone,

I have been working on a new version of Stride Docs for the Stride Game Engine (https://www.stride3d.net/) and would like to showcase the progress I've made. Although it's still a work in progress, there's enough to show how the docs can look nice using the `modern` template from DocFX, which utilizes Bootstrap 5. I've also incorporated emojis for added visual appeal.

I'd appreciate any comments or suggestions you may have. Thank you to everyone who is contributing to docfx.

Check out the old version here: https://doc.stride3d.net/latest/en/

And the new version here: https://stride-docs.vaclavelias.com/index.html

Here are some screenshots:

<img width="954" alt="image" src="https://user-images.githubusercontent.com/4528464/234119577-a5638832-d6df-4aef-9c67-173db71193ff.png">

<img width="940" alt="image" src="https://user-images.githubusercontent.com/4528464/234119684-52a2e50e-ac20-4509-8790-e7397e2c41f0.png">

Thank you 🙂

**Comment by yufeih** [comment]
This looks great!
